scripts/sitemap.ts

```diff
@@ -1,4 +1,4 @@
-import { generateSitemap } from 'pliny/utils/generate-sitemap.js';
+import { generateSitemap } from '../lib/utils/generate-sitemap.js';
```

Comment:
```suggestion
import { generateSitemap } from '@/lib/utils/generate-sitemap.js';
```
Shouldn't it be this?

Reply:
Yup, it can now! It was previously using `esbuild-register` which, for some reason I didn't understand wasn't picking up the path mappings.